src/main/java/racinggame/domain/Car.java

```diff
@@ -0,0 +1,70 @@
+package racinggame.domain;
+
+import static racinggame.view.OutputView.*;
+
+import java.util.Objects;
+
+public class Car {
+
+    public static final int MIN_MOVE_NUMBER = 0;
+    public static final int MAX_MOVE_NUMBER = 9;
+
+    private CarPosition position;
+    private final CarName name;
+
+    public Car(String name) {
+        this.name = new CarName(name);
+        this.position = new CarPosition();
+    }
+
+    public Car(String name, int position) {
+        this(name);
+        this.position = new CarPosition(position);
+    }
```

Comment:
```suggestion
    public Car(String name) {
        this(new CarName(name), new CarPosition(0));
    }

    public Car(String name, int position) {
        this(new CarName(name), new CarPosition(position));
    }
    
    public Car(CarName name, CarPosition position) {
        this.name = name;
        this.position = position;
    }
```
- 객체의 필드 값과 같은 타입의 주 생성자를 추가 
- 필드 값을 할당하는 작업은 주 생성자에서만
- 부 생성자는 주 생성자를 통해서만 필드 값을 초기화

생성자는 위와 같은 기준으로 구현할 것을 추천